db/range_tombstone_fragmenter.cc

```diff
@@ -0,0 +1,223 @@
+//  Copyright (c) 2018-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include "db/range_tombstone_fragmenter.h"
+
+#include <set>
```

Comment:
we usually alphabetize imports. I wonder if `make format` does this automatically.

Reply:
It does do that; I didn't run it earlier since I usually run `clang-format` from my editor, but it looks good now. I'll get into the habit of running `make format` before sending out PRs.